rust/gui-client/src-tauri/src/client/network_changes/windows.rs

```diff
@@ -188,6 +162,8 @@ pub fn check_internet() -> Result<bool> {
 /// Worker thread that can be joined explicitly, and joins on Drop
 pub(crate) struct Worker {
     inner: Option<WorkerInner>,
+    // I think this is `Arc` because of the worker thread, and we need the worker thread
+    // because of COM. `Notify` only has `&self` methods so it does not need `Arc` in general.
```

Comment:
It gets shared with another thread. But I have to replace it with an `mpsc` channel anyway per #4226, and when that happens I can remove the `Arc` because it'll be two separate tx and rx pieces.

Reply:
#4227 removes that `Arc`